android/src/main/java/cl/json/RNShareFileProvider.java

```diff
@@ -0,0 +1,7 @@
+package cl.json;
+
+import android.support.v4.content.FileProvider;
+
+public class RNShareFileProvider extends FileProvider {
```

Comment:
This is correct? This class extends FileProvider but dont seems to be used on another place.

Reply:
It is used in the manifest. Android says we cannot declare multiple providers with the same name. So we had to implement a class which extends FileProvider and then declare provider in the manifest using this class

Reply:
That is true and what makes solving FileProvider configuration so difficult, I hadn't thought to simply extend to work around it, makes sense now that I think about it.